### PR TITLE
Add Traditional Chinese language

### PR DESCRIPTION
**Pull request by MHNightCat**

### CODE REVIEW

**Comment by Miodec** [comment]
Have you verified you can type these words correctly? I know the input system had some issues with Chineese

**Comment by MHNightCat** [comment]
> Have you verified you can type these words correctly? I know the input system had some issues with Chineese

I don't know how to test

**Comment by Miodec** [comment]
Run the website locally or put the words into custom mode on monkeytype.com

**Comment by MHNightCat** [comment]
> 在本地運行網站或在 monkeytype.com 上將文字置於自定義模式

I tried testing it on the site and I think it's ok

**Comment by MHNightCat** [comment]
I think some people have problems with Chinese input because of the input method (there are many Chinese input methods)
But it's fine if I try it myself

File: frontend/static/languages/traditional_chinese.json [hunk 1]
```diff
@@ -0,0 +1,284 @@
+{
+    "name": "traditional_chinese",
```
Comment (Miodec) [inline frontend/static/languages/traditional_chinese.json:2]: Change this and the name of the file to `chinese_traditional`

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/languages/traditional_chinese.json (hunk 1 above)
Comment (MHNightCat) [reply frontend/static/languages/traditional_chinese.json]: ok done it

**Review by Miodec — APPROVED** [review]

**Comment by JacoChong** [comment]
MHNightCat I wanted to publicly thank you on discord but I couldn't find you. I have been holding my desire to have the Chinese language on mt, but there wasn't one and I couldn't help build one cuz of my baby level of programming. I instead added a Jyutping version (as in typing out the pronunciation notation of Cantonese) onto the website.

The server is down now and I couldn't try it out after a long inactivity on mt. Though it's probably using Bopomofo (sorry for assuming you are from Taiwan if it isn't), I am really really glad the day of finally having a legit Chinese language on the website actually came, and I can't wait to see other variations of Chinese languages (such as Cantonese) and input methods for them become available.

There are some testing sites on the web for Chinese created by other Taiwanese but those aren't as high-quality as mt, but because of your time devoted to making this work, I believe that the chinese language community on this website will slowly grow.

Once again thank you so so much for marking this big step for the Chinese language (at least for me it really is). I can't wait any longer to try it out even. THANK YOUUUUUU!!!!

**Pull request merged**